### PR TITLE
Pittsburgh Compound B

### PR DESCRIPTION
**Pull request by eapapp**
Hi @lzehl, here comes the updated PR for Pittsburgh Compound B. Can we keep the file name / instance ID as this readable name, or does it have to be the IUPAC name? (it may be hard to find in the KGE, I think)

### CODE REVIEW

File: instances/latest/terminologies/molecularEntity/pittsburghCompoundB.jsonld [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "synonym": [
+    "Pittsburgh Compound B",
+    "Pittsburgh B",
+    "PiB",
+    "benzothiazole-1",
+    "BTA-1",
+    "2-{4-[(11C)Methylamino]phenyl}-1,3-benzothiazol-6-ol",
+    "(N-methyl-(11c))2-(4'-methylaminophenyl)-6-hydroxybenzothiazole"
+  ]
+}
```
[the remark this answers is not in the document]
Comment (eapapp) [reply instances/latest/terminologies/molecularEntity/pittsburghCompoundB.jsonld]: Done ✅

**Comment by lzehl** [comment]
> Hi @lzehl, here comes the updated PR for Pittsburgh Compound B. Can we keep the file name / instance ID as this readable name, or does it have to be the IUPAC name? (it may be hard to find in the KGE, I think)

For the EBRAINS KG we need to bring up that the synonyms should become searchable within the KGE and potentially also for the KGS.

File: instances/latest/terminologies/molecularEntity/pittsburghCompoundB.jsonld [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/pittsburghCompoundB",
```
[the remark this answers is not in the document]
Comment (eapapp) [reply instances/latest/terminologies/molecularEntity/pittsburghCompoundB.jsonld]: Done ✅

File: instances/latest/terminologies/molecularEntity/pittsburghCompoundB.jsonld [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/pittsburghCompoundB",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "Pittsburgh compound B (PiB) is a radioactive analog of thioflavin T, which can be used in positron emission tomography scans to image beta-amyloid plaques in neuronal tissue. Due to this property, Pittsburgh compound B may be used in investigational studies of Alzheimer's disease.",
```
[the remark this answers is not in the document]
Comment (lzehl) [reply instances/latest/terminologies/molecularEntity/pittsburghCompoundB.jsonld]: feel free to further modify but I would leave out usage. we can work on something more detailed for the description

**Comment by lzehl** [comment]
@eapapp should I rereview?

**Comment by eapapp** [comment]
It seems I added the file to the wrong place 🙈 It is in latest and not v3.0. Sorry for that. It is probably best if you check whether the file contents look okay before we copy it anywhere.

**Comment by lzehl** [comment]
> It seems I added the file to the wrong place 🙈 It is in latest and not v3.0. Sorry for that. It is probably best if you check whether the file contents look okay before we copy it anywhere.

okay.

**Comment by lzehl** [comment]
@eapapp I moved the instance you wrote now to your v3.0, and added it to v4.0 and latest in your branch with the respective changes. @Raphael-Gazzotti could you do a sanity check and merge if everything is okay now ?

**Comment by lzehl** [comment]
@eapapp this will get easier in future: on the one hand we are working on a validation on the PRs providing feedback on formal errors in instances and on the other hand we will have a pipeline that will auto-populate an instance across versions (so that the PR can be made for one version only without the extra effort)

**Review by Raphael-Gazzotti — APPROVED** [review]

**Pull request merged**